slate-docs/source/includes/_main-generator.md

```diff
@@ -108,33 +108,41 @@ configBuilder.without(
       <td>Replacing encountered <code>Optional</code> instances as null-able forms of their generic parameter type.</td>
       <td>-</td>
     </tr>
-    <tr><th>#</th><th>Behavior if included</th><th>Behavior if excluded</th></tr>
```

Comment:
Let's keep the same number of `Option`s between those repetitions of the table header.